src/azul/plugins/repository/canned/__init__.py

```diff
@@ -113,7 +113,7 @@ def list_bundles(self, source: CannedSourceRef, prefix: str) -> List[CannedBundl
     def fetch_bundle(self, bundle_fqid: CannedBundleFQID) -> Bundle:
         self._assert_source(bundle_fqid.source)
         now = time.time()
-        staging_area = self.staging_area(bundle_fqid.source.name)
+        staging_area = self.staging_area(bundle_fqid.source.spec)
         version, manifest, metadata = staging_area.get_bundle_metadata(bundle_fqid.uuid)
```

Comment:
```suggestion
        version, manifest, metadata = staging_area.get_bundle(bundle_fqid.uuid)
```
The name of this function changed when `staging_area.py` was checked into `hca-metadata-api`. Please add a separate commit to make this typo fix.